doc/styles/TextBox.md

```diff
@@ -87,3 +88,7 @@ IsDefaultStyle\*: Styles in this column will be set as the default implicit styl
 | `MaterialOutlinedTextBoxMinHeight`                | `Double`          | 56                            |
 | `MaterialOutlinedTextBoxBorderPadding`            | `Thickness`       | 1                             |
 | `MaterialOutlinedTextBoxBorderThicknessFocused`   | `Double`          | 2                             |
+| `MaterialOutlinedTextBoxPaddingFocused`           | `Thickness`       | 0                             |
+| `MaterialOutlinedTextBoxPaddingPointerOver`       | `Thickness`       | 0                             |
+| `MaterialOutlinedTextBoxBorderThicknessPointerOver`| `Double`         | 2                             |
+| `MaterialFilledTextBoxBorderHeightNormal`         | `Double`          | 1                             |
```

Comment:
```suggestion
| `MaterialFilledTextBoxBorderHeightNormal`         | `Double`          | 1                             |

```

For the current markdown error for the CI Build:
doc/styles/TextBox.md:94:105 MD047/single-trailing-newline Files should end with a single newline character